applications/StructuralMechanicsApplication/tests/cpp_tests/test_mpc_plate_tension.cpp

```diff
@@ -0,0 +1,178 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:             BSD License
+//                                       license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:    Yaman Zendaki , Hoang-Giang Bui
+//
+
+// System includes
+#include <iostream>
+// External includes
+
+// Project includes
+//#include "includes/gid_io.h"
+#include "testing/testing.h"
+#include "containers/model.h"
+#include "includes/model_part.h"
+#include "spaces/ublas_space.h"
+#include "includes/linear_master_slave_constraint.h"
+#include "structural_mechanics_application.h"
+#include "custom_constitutive/linear_plane_strain.h"
+#include "utilities/variable_utils.h"
+// Linear solvers
+#include "linear_solvers/reorderer.h"
+#include "linear_solvers/direct_solver.h"
+#include "linear_solvers/linear_solver.h"
```

Comment:
please clean your includes, there are a few you don't need

Reply:
Thank you for the review and information
all above mentioned issues have been solved in c92174185cb83cfc3baf170babeb4a740567bc1c

Reply:
thanks @yamanatoo for your contribution